test/e2e/cluster_config.sh

```diff
@@ -47,6 +47,11 @@ clusters:
     karpenter_pools_enabled: "true"
     okta_auth_client_id: "kubernetes.cluster.teapot-e2e"
     teapot_admission_controller_validate_pod_images_soft_fail_namespaces: "^kube-system$"
+    skipper_open_policy_agent_enabled: "true"
```

Comment:
We should discuss if we need a negative test, i.e. verify that clusters without OPA can still be deployed. We might otherwise not catch config errors where without this flag the yaml is invalid.

Reply:
@mikkeloscar is this something that can be done here?

AFAIU within these e2e testing, it pre-configures one cluster and do validations on that. Changing the base cluster configurations during the test might not be possible. May be there is a different location where we can test different config flags.

Reply:
We currently don't have the capability to test against different permutations of configuration.